### ISSUE TITLE
What is it like to ...

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/nelsonic.github.io#290]
what is it like to work for Mark Cuban?
https://youtu.be/A2kjYnmFv0U?t=40m31s

**Comment by iteles** [comment]
> I micro-manage until I trust people and then I expect weekly reports

> And start with the bad news, I expect you to do well, so I want to know what went badly and why

:+1:

**Comment by nelsonic** [comment]
@iteles thought you might find that _amusing_... 😉 
"_micro-management_" is **never** "***pleasant***" for _either_ person, 
but _sometimes_ (_at the very beginning_) it _ca be **necessary**_... 🤔

**Comment by iteles** [comment]
Although it has to be contextualised. If not, you run the risk of 'knocking the initiative out of' people and creating a dependency on the micro-management 🙃

**Comment by nelsonic** [comment]
Yeah, I think the _only_ way is for the _first_ person who _does_ a task to write down the _process_. 📝 
Then anyone `else` who comes next can _easily_ follow it, iterate & improve. 🚀
